.github/workflows/build.yml

```diff
@@ -39,13 +42,64 @@ jobs:
         make -f certifier_tests.mak clean
 
         make -j2 -f certifier.mak
+
+        # We need to clean here, otherwise make certifier_tests.mak will run
+        # into some protobuf-related errors. But this 'clean' will also
+        # rm ../libcertifier_framework.so, so it will need to be re-built.)
         make -f certifier.mak clean
 
         make -j2 -f certifier_tests.mak
         ./certifier_tests.exe
 
+        # Re-build ...; otherwise pylint will complain below.
+        make -f certifier.mak ../libcertifier_framework.so
+
+        popd
+
+        #! Sanity check that the shared library was built correctly and can be loaded
+        #! python3 -c "import libcertifier_framework; help(libcertifier_framework)"
+        #! python3 -c "import libcertifier_tests; help(libcertifier_tests)"
+
+    #! -------------------------------------------------------------------------
+    #! Code formatting runs pylint, so this step can only come after test-py-bindings
+    #! where the shared library (imported by Pytests code) is built.
+    #! -------------------------------------------------------------------------
+    - name: test-code-formatting
+      run: |
+        ./CI/scripts/check-gofmt.sh
+
+        pushd tests
+
+        set -x
+        pylint *.py
+        set +x
+
+        popd
+
+    #! -------------------------------------------------------------------------
+    - name: test-cert_framework-pytests
```

Comment:
See the output for this chunk `test-cert_framework-pytests` in the CI runs, to get an idea exactly-how these pytest tests run, and what outputs they will create.

Reply:
Update: In final version, relocated new Python tests to `tests/pytests` sub-dir and adjusted commands in build.yml that run `pylint` or execute `pytest`.